code/Base/GamemodeSystem/Entities/BaseGamemode.TargetEntities.cs

```diff
@@ -0,0 +1,44 @@
+namespace Sports;
+
+public partial class BaseGamemode
+{
+	public List<Entity> Entities { get; protected set; } = new();
+
+	public IEnumerable<Entity> GetEntitiesWithTag( string tag )
+	{
+		return Entities.Where( x => x.Tags.Has( tag ) );
+	}
+	public IEnumerable<T> GetEntitiesOfType<T>( string tag = null )
+	{
+		if ( tag is not null )
```

Comment:
The logic here seems to be the wrong way around?

Reply:
Yeah, whoops